packages/js/product-editor/src/hooks/use-variations-order.ts

```diff
@@ -0,0 +1,93 @@
+/**
```

Comment:
Curious why you only copied this one over, but left the original hook within the `client` code?

Reply:
Just spaced the clean-up on that one, fixed in https://github.com/woocommerce/woocommerce/pull/37006/commits/1f65faf0ab8f7d5f9eb293acd1313990ee5e7bbf